Cargo.toml

```diff
@@ -10,10 +10,12 @@ homepage = "https://github.com/elkowar/eww"
 
 
 [features]
-default = ["x11"]
+default = ["wayland"]
```

Comment:
No.

Default should still be x11, as _most_ people are on X11.

Reply:
I'll one day learn how to use flags.

Reply:
--no-default-features --features=wayland